app/views/mailers/tasks/task_completed.html.slim

```diff
@@ -0,0 +1,36 @@
+table.email-table
```

Comment:
tables a heavy.. since we adding a stylesheet - could you pls make template in a `div` style